apps/hash-frontend/src/pages/notes.page/today-section.tsx

```diff
@@ -0,0 +1,184 @@
+      if (!quickNoteEntities || typeof quickNotesSubgraph === "undefined") {
+        return undefined;
+      }
+
+      const { quickNoteEntity: latestQuickNoteEntity } =
+        quickNoteEntities[0] ?? {};
+
+      if (!latestQuickNoteEntity || !quickNotesSubgraph) {
+        return null;
+      }
+
+      const contents = getBlockCollectionContents({
+        blockCollectionEntityId:
+          latestQuickNoteEntity.metadata.recordId.entityId,
+        blockCollectionSubgraph: quickNotesSubgraph,
+      });
+
+      if (isBlockCollectionContentsEmpty({ contents })) {
+        return latestQuickNoteEntity;
+      }
+      return null;
+    }, [quickNoteEntities, quickNotesSubgraph]);
+
+    const displayedQuickNoteEntities = useMemo(
+      () =>
+        quickNoteEntities?.filter(
+          ({ quickNoteEntity }) =>
+            (!creatingQuickNote ||
+              quickNoteEntity.metadata.recordId.entityId !==
+                creatingQuickNote.metadata.recordId.entityId) &&
+            (!latestQuickNoteEntityWithEmptyContents ||
+              latestQuickNoteEntityWithEmptyContents.metadata.recordId
+                .entityId !== quickNoteEntity.metadata.recordId.entityId),
+        ),
+      [
+        quickNoteEntities,
+        creatingQuickNote,
+        latestQuickNoteEntityWithEmptyContents,
+      ],
+    );
+
+    return (
+      <NotesSectionWrapper ref={ref}>
+        <TimestampColumn
+          heading="Today"
+          subheading={format(new Date(), "yyyy-MM-dd")}
+          isCollapsed={isCollapsed}
+          toggleIsCollapsed={() => setIsCollapsed(!isCollapsed)}
+          navigateDown={navigateDown}
+        />
+        <Box flexGrow={1}>
+          <NotesWrapper sx={{ padding: ({ spacing }) => spacing(3.25, 4.5) }}>
+            {/* 
+            If the last created quick note is empty, we re-use it to populate the
+            create quick note form. This prevents the quick notes page from creating
+            a new quick note on every page load.
+
+            @todo: when we have draft entities, we could use a draft quick note to
+            populate this input instead. Alternatively, we could refactor the
+            `BlockCollection` component to support rendering the input without
+            a block collection entity having been persisted yet.
+            */}
+            <CreateQuickNote
+              initialQuickNoteEntity={latestQuickNoteEntityWithEmptyContents}
+              initialQuickNoteEntitySubgraph={
+                latestQuickNoteEntityWithEmptyContents && quickNotesSubgraph
+                  ? quickNotesSubgraph
+                  : undefined
+              }
+              refetchQuickNotes={refetchQuickNotes}
+              onCreatingQuickNote={setCreatingQuickNote}
+            />
+            <Divider sx={{ borderColor: ({ palette }) => palette.gray[20] }} />
+            <Box display="flex" marginTop={2.25}>
+              <Box display="flex" alignItems="center" gap={1.5}>
```

Comment:
I suggest commenting this out until it does something specific to the note.. at the moment it's a bit weird because they are just links to create a new entity/type with no relation to the note being typed. Mentioned this to @nonparibus who is happy we hide this until it makes sense

Reply:
Alright, have uncommented it for now in https://github.com/hashintel/hash/pull/3434/commits/f576ac5bba2539a3204ac3444beb694bae7f7663